lib/airbrakex/notifier.ex

```diff
@@ -36,11 +36,14 @@ defmodule Airbrakex.Notifier do
   defp add_error(payload, nil), do: payload
 
   defp add_error(payload, error) do
-    error = Map.from_struct(error)
+    error = get_error_as_map(error)
```

Comment:
If this fix is being implemented elsewhere, I can remove it from this PR as well.